app/build.gradle

```diff
@@ -75,13 +75,15 @@ dependencies {
     implementation 'androidx.preference:preference-ktx:1.2.0'
```

Comment:
@cyb3rko Do you mind having a look at this PR? You've done significant changes to gotify/android and your input would be helpful.

No pressure if you don't have time for this (:.